components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -51,10 +51,10 @@ const useStyles = makeStyles((theme: Theme) =>
                 theme.palette.type === 'light' ? theme.palette.secondary[50] : theme.palette.secondary.light,
             borderRadius: '0px 24px 24px 0px',
             opacity: 0.9,
-            '&.rectangular': {
-                width: '100%',
-                borderRadius: 0,
-            },
+        },
+        activeSquare: {
+            width: '100%',
+            borderRadius: 0,
         },
```

Comment:
if you want, you can still do these nested like you originally had. The format (using a JSS class instead of a regular string class):
```
        active: {
            // ...regular rules
            // with additional class
            '& $square': {
                // square rules
            },
        },
        // these must be defined, even if empty so we can reference them in other nested rules
        square: {},
```

Reply:
Does not work...